### DISCUSSION TITLE
这个是这么实现分布式的啊

### DISCUSSION THREAD

**Discussion by skyformat99** [nanomq/nanomq#461] [General]
从图上看，似乎支持分布式
但是，具体是如何实现的呢？
能不能讲一讲原理

**Comment by JaylinYu** [comment]
方案介绍中是建议使用MQTT桥接的方式进行边缘多节点间的数据流转，并非传统意义上的分布式。
分布式+MESH组网尚未确定支持，依未来实际使用场景我们会考虑开发的。
